### PR TITLE
[prep] ostree_repo_new_at()

### PR DESCRIPTION
**Pull request by cgwalters**
Prep for https://github.com/ostreedev/ostree/pull/820

### CODE REVIEW

**Comment by cgwalters** [comment]
This one just flaked, should be good to go.

File: src/libostree/ostree-sysroot.c [hunk 1]
```diff
@@ -729,6 +729,20 @@ ostree_sysroot_load (OstreeSysroot  *self,
...
+static gboolean
+ensure_repo_opened (OstreeSysroot  *self,
+                    GError        **error)
+{
+  if (self->repo_opened)
+    return TRUE;
+  if (!ostree_repo_open (self->repo, NULL, error))
+    return FALSE;
+  /* And flag it as a system repo */
+  self->repo->is_system = TRUE;
```
Comment (jlebon) [inline src/libostree/ostree-sysroot.c]: This should already be `TRUE` at the time the sysroot was constructed, right?

Comment (cgwalters) [inline src/libostree/ostree-sysroot.c]: Yeah, I removed this.  It'll come back though in the later patches when this bit moves around.

**Comment by jlebon** [comment]
@rh-atomic-bot r+ 5831800

**Pull request closed without merging**